docs/source/buildling.rst

```diff
@@ -0,0 +1,160 @@
+.. _building:
+
+Building PYJNIus
+================
+
+Building PyJNIus depends on `Cython <http://cython.org/>`_ and the `Java
+<http://www.oracle.com/javase>`_ Development Kit (includes the Java Runtime
+Environment).
+
+
+Installation on GNU/Linux distributions
```

Comment:
[Here and throughout] The subsections are titled "Installation" but should be titled "Building"